torch/_torch_docs.py

```diff
@@ -2637,124 +2637,121 @@ def merge_dicts(*dicts):
 add_docstr(
     torch.cholesky_solve,
     r"""
-cholesky_solve(input, input2, upper=False, *, out=None) -> Tensor
+cholesky_solve(B, L, upper=False, *, out=None) -> Tensor
 
-Solves a linear system of equations with a positive semidefinite
-matrix to be inverted given its Cholesky factor matrix :math:`u`.
+Solves a square linear system of equations with a positive semidefinite
+matrix :math:`A` to be inverted given its Cholesky factor matrix :math:`L`.
```

Comment:
Nit. No need to mention A and L here. A pattern we used in linalg was not to use math here, as this is the paragraph that's rendered at https://pytorch.org/docs/stable/linalg.html. What about
```suggestion
Computes the solution of a system of linear equations with complex Hermitian or real symmetric positive-definite lhs given its Cholesky decomposition.
```
Note that there was also a small typo: semidefinite -> definite